### DISCUSSION TITLE
convertFilter export/import

### DISCUSSION THREAD

**Discussion by DanielForniessoria-TomTom** [maplibre/maplibre-gl-js#2120] [Q&A]
I found the issue when trying to import "convertFilter" from maplibre-gl. It doesn't seem to be "cleanly" exported, and thus can't be used properly. 
The import I can use is
`import { convertFilter } from "maplibre-gl/dist/style-spec";`
However, when packing my own library, to then use it by another app, that other app gives an error:
`Module not found: Error: Can't resolve 'maplibre-gl/dist/style-spec' in '/Users/dafo/projects/go-sdk-js/map/dist'`

I wonder if others have also found a similar issue. Am I missing something, or should we add a fix for this in MapLibre?

**Comment by HarelM** [comment]
You can install and import from [@maplibre/maplibre-gl-style-spec](https://www.npmjs.com/package/@maplibre/maplibre-gl-style-spec)
Importing from the dist folder using typescript is not recommended (and can probably cause these issues).
